environments/staging/vars.yml

```diff
@@ -12,6 +12,8 @@ superset_db_name: superset_meta
 oauth2_db_name: superset_oauth2
 cchq_db_name: superset_cchq_data
 
+rabbitmq_version: 3.9.13-1
```

Comment:
Any specific reason for not using the latest version of `rabbitmq`? https://www.rabbitmq.com/release-information

Reply:
I took the one that was showing up in the available versions for the server I am installing this on, `22.04.4 LTS`.
I see HQ is using `3.10.13-1`, So I will be okay to jump to that at least, if its available.

Reply:
If this is a fresh install, I would suggest go with the latest supported version, which I believe is 4. This would be helpful in longer run where you will save some time in atleast one of the upgrades.

Reply:
It appears the version being used by HQ is the latest one available via apt. Anything above that version gives a long error message complaining about many depenedency though mainly probably with erlang version.
Looks like we need to change how we install erlang.
```
The following information may help to resolve the situation:
The following packages have unmet dependencies:
rabbitmq-server : Depends: erlang-base (>= 1:24.3.2) but 1:24.2.1+dfsg-1ubuntu0.1 is to be installed or
                            erlang-base-hipe (>= 1:24.3.2) but it is not installable or
                            esl-erlang (>= 1:24.3.2) but it is not installable
                   Depends: erlang-crypto (>= 1:24.3.2) but 1:24.2.1+dfsg-1ubuntu0.1 is to be installed or
                           esl-erlang (>= 1:24.3.2) but it is not installable
                   Depends: erlang-eldap (>= 1:24.3.2) but 1:24.2.1+dfsg-1ubuntu0.1 is to be installed or
                            esl-erlang (>= 1:24.3.2) but it is not installable
                   Depends: erlang-inets (>= 1:24.3.2) but 1:24.2.1+dfsg-1ubuntu0.1 is to be installed or
...         
```
So, using the same version as on HQ which I can update to without breaking anything else.